fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,8 +1,14 @@
-Dhizuku is a tool that allows you to share device owner permissions with other apps.
+<i>Dhizuku</i> is a tool that allows you to share device owner permissions with other apps.
 
-<b>Enable service</b>
+This app is similar to Shizuku (ADB) and Magisk (Root) but instead, it shares device owner permissions with other apps. An example use would be having a new Installer in the Aurora Store for allowing devices such as those sold in Japan (and those where normal package installation is blocked) to install apps from third-party app stores.
```

Comment:
Looks oddly similar to my comment on RFP on F-Droid. Please don't use my comment here especially mentioning Aurora Store. Both projects are unrelated.

Reply:
So, first of all, Shizuku and Dhizuku are completely different things. Please focus on each one individually.
Shizuku is for sharing internal shell services, while Dhizuku has no such functionality and is for sharing device owner permissions.

A Device Owner has higher privileges than a Device Admin and can only have one per device.
And you'll need Dhizuku to use the Device Owner across multiple apps.

To explore the features and benefits of Device Owner, try [**TestDPC**](https://github.com/googlesamples/android-testdpc).

Reply:
> Please don't use my comment here especially mentioning Aurora Store. Both projects are unrelated.

That was my bad then. I've integrated that with the app description to make it a bit clearer what the app is intended for, which was hard to figure by "outsiders" not involved with developing the app or having deep knowledge on the topic otherwise.

Once a "proper" description has been merged to the fastlane structures here, please give me a ping so I enable that on my end and also manually trigger an update for it to be reflected in the IzzyOnDroid repo. Thanks a lot!